algorithms/fast_and_slow/find_duplicate/README.md

```
@@ -37,3 +37,178 @@ Explanation 3:
+With this positioning and pace of pointers, pointers are guaranteed to meet at the cycle’s starting point, corresponding
+to the duplicate number.
+
+Now, let’s look at the detailed workflow of the solution:
+
+For this problem, the duplicate number will create a cycle in the nums array. The cycle in the nums array helps identify
+the duplicate number.
+
+To find the cycle, we’ll move in the nums array using the `f(x)=nums[x]`, where x is the index of the array. This
+function constructs the following sequence to move:
+
+      x, nums[x], nums[nums[x]], nums[nums[nums[x]]], ...
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)